src/scenes/home/home.js

```diff
@@ -61,69 +61,71 @@ class Home extends Component {
   setBgImage(location) {
     if (location.pathname === '/') {
       this.setState({
-        bgChanged: !(this.state.bgImage),
+        bgChanged: !this.state.bgImage,
         bgImage: true,
         bgImageUrl: familyImage,
-        bgImageStyle: "backgroundImageHome"
+        bgImageStyle: 'backgroundImageHome',
       });
     } else if (location.pathname === '/team') {
       this.setState({
-        bgChanged: !(this.state.bgImage),
+        bgChanged: !this.state.bgImage,
         bgImage: true,
         bgImageUrl: lincolnImage,
-        bgImageStyle: "backgroundImageTeam"
+        bgImageStyle: 'backgroundImageTeam',
       });
     } else if (location.pathname === '/history') {
       this.setState({
-        bgChanged: !(this.state.bgImage),
+        bgChanged: !this.state.bgImage,
         bgImage: true,
         bgImageUrl: colinPowellImage,
-        bgImageStyle: "backgroundImageTeam"
+        bgImageStyle: 'backgroundImageTeam',
       });
     } else {
       this.setState({
         bgChanged: this.state.bgImage,
         bgImage: false,
         bgImageUrl: null,
-        bgImageStyle: null
+        bgImageStyle: null,
       });
     }
   }
 
-  updateRootAuthState = (cb) => {
+  updateRootAuthState = cb => {
```

Comment:
Unsure what `cb` is, and it's bothered me on other PRs. Can you mind renaming the variable to something more descriptive?

Unrelated to the actual PR, but that's it - everything else is good!

Reply:
@kylemh `cb` frequently stands for 'callback' - in this case, it would be an optional callback function passed to `updateRootAuthState` that could use `this.props.history` in some way.